test/network_interface/ApolloNetworkInterface.ts

```diff
@@ -307,3 +317,56 @@ describe('PersistedQueryNetworkInterface', () => {
     });
   });
 });
+
+describe('addPersistedQueries', () => {
+  class GenericNetworkInterface implements NetworkInterface {
+    public query(originalQuery: Request) {
+      return Promise.resolve(originalQuery);
```

Comment:
I'm not sure how this passes the type checking, but this Promise should be resolved with an `ExecutionResult`